src/Microsoft.AspNet.Razor.Runtime/Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -135,7 +135,7 @@ public bool ChildContentRetrieved
         /// <summary>
         /// <see cref="ITagHelper"/>s that should be run.
         /// </summary>
-        public IEnumerable<ITagHelper> TagHelpers
+        public IList<ITagHelper> TagHelpers
```

Comment:
The general principle should be to expose what's required and this doesn't do that -- the only reader needs a sorted read-only list and doesn't get one. But it's probably not that big a deal because the `TagHelperExecutionContext` isn't exposed to tag helper authors (or most other user code) directly.